glossary.tex

```diff
@@ -1,31 +1,31 @@
-\newglossaryentry{Cluster}
+\newglossaryentry{cluster}
 {
-  name={Cluster},
+  name={cluster},
   description={A group of compute nodes},
 }
-\newglossaryentry{Compute-Node}
+\newglossaryentry{compute-node}
 {
-  name={Compute Node},
+  name={compute node},
   description={The computational units on which batch or interactive jobs are processed. A compute node is pretty much comparable to a single personal computer. It contains one or more sockets, each holding a single CPU. Some nodes also contain one or more GPGPUs. The compute node is equipped with memory (RAM) that is accessible by all its CPUs},
```

Comment:
We also call this `workernode` or simply `node`, so please add an entry for both and refer to this one.